tests/component/sections/collection/Collection.spec.tsx

```diff
@@ -21,6 +21,11 @@ describe('Collection page', () => {
   })
 
   it('renders skeleton while loading', () => {
+    const DELAYED_TIME = 200
```

Comment:
Just curious if the change in this skeleton test is related to the lazy loading?  Or just to make sure that the timing of the test is more deterministic?

Reply:
I've always had the feeling that these tests of checking loading skeletons were a bit flaky tests.
This makes the test more robust, I found this solution to delay a stub while working on the collection page and felt it was appropriate to implement them in these tests.